lib/routes/check-in/index.js

```diff
@@ -3,6 +3,18 @@ const schema = require('./schema')
 const { TooManyRequests } = require('http-errors')
 const { checkInInsert } = require('./query')
 
+/**
+ * Allows users to submit a daily health checkin. All application users can perform a daily checkin
```

Comment:
Note the symptom checker in the app is an optional piece of functionality

Reply:
Removed mention of what the App allows a user to do and focuses on what this endpoint supports.